src/core/bonded_interactions/bonded_interactions.dox

```diff
@@ -503,3 +533,58 @@
  *  @f$ \theta_{ijk} = \pi @f$.
  *
  */
+
+/*
```

Comment:
This break transformed the next block into a regular comment instead of a doxygen comment.
```suggestion
 *
```